tests/test_writer.py

```diff
@@ -249,3 +249,109 @@ def test_dataset_iter_determinism(self, local_remote_dir: Tuple[str, str], num_s
         # Ensure sample iterator is deterministic
```

Comment:
Same here. Just test the `bytes_to_int()` and `number_abbrev_to_int()` with various parameter as part of test_util.py !!